metriken-exposition/src/snapshotter.rs

```diff
@@ -69,6 +69,7 @@ impl Snapshotter {
                 Some(Value::Counter(value)) => {
                     let counter = Counter {
                         name: metric.formatted(metriken::Format::Simple),
+                        description: metric.description().map(|v| v.to_string()),
```

Comment:
Is there any advantage to keeping this as a separate field vs inserting it into the metadata right here and then avoiding any changes to snapshot.rs and parquet.rs?

Reply:
Per discussion, I've made the change and reverted the changes to snapshot.rs and parquet.rs